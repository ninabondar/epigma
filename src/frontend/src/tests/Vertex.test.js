import React from "react"
import { createPoint } from "../utils/helper"
import Vertex from "../components/Vertex"

describe("<Vertex/>", () => {
  const point = createPoint({ x: 0, y: 0 })

  it("should match its snapshot", () => {
    const component = shallow(<Vertex point={point}/>)
    expect(component).toMatchSnapshot()
  })
})