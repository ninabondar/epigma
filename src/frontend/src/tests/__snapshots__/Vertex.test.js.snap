// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Vertex/> should match its snapshot 1`] = `
<circle
  className="Vertex"
  cx={0}
  cy={0}
  onMouseDown={[Function]}
  r={4}
/>
`;
